content/en/blog/_posts/2020-03-25-kubernetes-1.18-release-announcement.md

```diff
@@ -0,0 +1,137 @@
+
+### SIG-CLI introduces kubectl debug
+
+SIG-CLI was debating the need for a debug utility for quite some time already. With the development of [ephemeral containers](https://kubernetes.io/docs/concepts/workloads/pods/ephemeral-containers/), it became more obvious how we can support developers with tooling built on top of `kubectl exec`. The addition of the [`kubectl debug` command](https://github.com/kubernetes/enhancements/blob/master/keps/sig-cli/20190805-kubectl-debug.md) (it is alpha but your feedback is more than welcome), allows developers to easily debug their Pods inside the cluster. We think this addition is invaluable.  This command allows one to create a temporary container which runs next to the Pod one is trying to examine, but also attaches to the console for interactive troubleshooting.
+
+
+## Introducing Windows CSI support alpha for Kubernetes
+
+With the release of Kubernetes 1.18, an alpha version of CSI Proxy for Windows is getting released. CSI proxy enables non-privileged (pre-approved) containers to perform privileged storage operations on Windows. CSI drivers can now be supported in Windows by leveraging CSI proxy.
+
+
+## Other Updates
+
+### Graduated to Stable 💯
+
+- [Taint Based Eviction](https://github.com/kubernetes/enhancements/issues/166)
+- [Kubectl Diff](https://github.com/kubernetes/enhancements/issues/491)
+- [CSI Block storage support](https://github.com/kubernetes/enhancements/issues/565)
+- [APIServer DryRun](https://github.com/kubernetes/enhancements/issues/576)
+- [Pass Pod information in CSI calls](https://github.com/kubernetes/enhancements/issues/603)
+- [Support Out-of-Tree vSphere Cloud Provider](https://github.com/kubernetes/enhancements/issues/670)
+- [Support GMSA for Windows workloads](https://github.com/kubernetes/enhancements/issues/689)
+- [Skip attach for non-attachable CSI volumes](https://github.com/kubernetes/enhancements/issues/770)
+- [Extend allowed PVC DataSources](https://github.com/kubernetes/enhancements/issues/989)
+- [Moving kubectl package code to staging](https://github.com/kubernetes/enhancements/issues/1020)
+- [RunAsUserName for Windows](https://github.com/kubernetes/enhancements/issues/1043)
+- [Adding AppProtocol to Services and Endpoints](https://github.com/kubernetes/enhancements/issues/1507)
+- [Extending Hugepage Feature](https://github.com/kubernetes/enhancements/issues/1539)
+- [client-go signature refactor to standardize options and context handling](https://github.com/kubernetes/enhancements/issues/1601)
+- [Graduate NodeLocal DNSCache to GA ](https://github.com/kubernetes/enhancements/issues/1024)
+
+
+### Major Changes
+
+- [EndpointSlice API](https://github.com/kubernetes/enhancements/issues/752)
+- [Moving kubectl package code to staging](https://github.com/kubernetes/enhancements/issues/1020)
+- [Graduate Ingress to V1](https://github.com/kubernetes/enhancements/issues/1453)
```

Comment:
This makes it look as if Ingress went GA, which I don't think is true.

Reply:
we're taking this out now